velox/connectors/hive/storage_adapters/abfs/tests/MockDataLakeFileClient.h

```diff
@@ -14,26 +14,36 @@
  * limitations under the License.
  */
 
-#include "velox/connectors/hive/storage_adapters/abfs/AbfsWriteFile.h"
-
 #include "velox/exec/tests/utils/TempFilePath.h"
 
-using namespace facebook::velox;
-using namespace facebook::velox::filesystems::abfs;
+#include "velox/connectors/hive/storage_adapters/abfs/AzureDataLakeFileClient.h"
+
+using namespace Azure::Storage::Files::DataLake::Models;
+
+namespace facebook::velox::filesystems {
 
-namespace facebook::velox::filesystems::test {
-// A mocked blob storage file client backend with local file store.
-class MockBlobStorageFileClient : public IBlobStorageFileClient {
+// A mock AzureDataLakeFileClient backend with local file store.
+class MockDataLakeFileClient : public AzureDataLakeFileClient {
  public:
-  MockBlobStorageFileClient() {
-    auto tempFile = ::exec::test::TempFilePath::create();
+  MockDataLakeFileClient() {
+    auto tempFile = velox::exec::test::TempFilePath::create();
     filePath_ = tempFile->getPath();
   }
 
+  MockDataLakeFileClient(std::string_view filePath) : filePath_(filePath) {}
+
+  std::string_view path() {
```

Comment:
Nit, this could be `std::string_view path() const`.

Reply:
done!